server/src/main/scala/net/wiringbits/actions/CreateUserAction.scala

```diff
@@ -41,16 +39,13 @@ class CreateUserAction @Inject() (
           hashedPassword = hashedPassword,
           verifyEmailToken = hmacToken
         )
-      _ <- usersRepository.create(createUser)
-
-      // then, send the verification email
       emailParameter = s"${createUser.id}_$token"
-      emailRequest = EmailMessage.registration(
+      emailMessage = EmailMessage.registration(
         name = createUser.name,
         url = webAppConfig.host,
         emailParameter = emailParameter
       )
-      _ = emailApi.sendEmail(EmailRequest(request.email, emailRequest))
+      _ <- usersRepository.create(createUser, emailMessage)
```

Comment:
We need to keep sending the email in this scenario, otherwise, the token will be logged in plain text in the database